src/microbit/__model/microbit_model.py

```diff
@@ -25,9 +28,21 @@ def running_time(self):
         print(f"time. time: {time.time()}")
         return time.time() - self.__start_time
 
+    def temperature(self):
+        return self.__temperature
+
+    def __set_temperature(self, temperature):
+        if (
+            temperature < CONSTANTS.MIN_TEMPERATURE
+            or temperature > CONSTANTS.MAX_TEMPERATURE
+        ):
+            raise ValueError(CONSTANTS.INVALID_TEMPERATURE_ERR)
+        else:
+            self.__temperature = temperature
+
     def update_state(self, new_state):
         for button_name in CONSTANTS.EXPECTED_INPUT_BUTTONS:
-            button = self.microbit_button_dict[button_name]
+            button = self.__microbit_button_dict[button_name]
 
             previous_pressed = button.is_pressed()
             button_pressed = new_state.get(button_name, previous_pressed)

```

Comment:
_nit_: change variable name to is_button_pressed to make it more obvious its a boolean. If you change that, maybe change the other name too. What do you think?